<ui version="4.0" >
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QFormLayout" name="formLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>&amp;Hostname</string>
           </property>
           <property name="buddy" >
            <cstring>hostLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="hostLineEdit" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>&amp;Database Name</string>
           </property>
           <property name="buddy" >
            <cstring>databaseLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="databaseLineEdit" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&amp;Driver</string>
           </property>
           <property name="buddy" >
            <cstring>driversComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="driversComboBox" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>&amp;Username</string>
           </property>
           <property name="buddy" >
            <cstring>userLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="userLineEdit" />
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>&amp;Password</string>
           </property>
           <property name="buddy" >
            <cstring>passwordLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QLineEdit" name="passwordLineEdit" />
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons" >
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>&amp;Connections</string>
         </property>
         <property name="buddy" >
          <cstring>connListWidget</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="connListWidget" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
